src/fetch_real_estate_prices.py

```diff
@@ -0,0 +1,10 @@
+from urllib.request import urlretrieve
+
+urls = [
+    ('region', 'http://seriesestatisticas.ibge.gov.br/exportador.aspx?arquivo=PRECO415_GR_ABS.csv&localidade=Todas'),
+    ('state', 'http://seriesestatisticas.ibge.gov.br/exportador.aspx?arquivo=PRECO415_UF_ABS.csv&localidade=Todas'),
+    ('brasil', 'http://seriesestatisticas.ibge.gov.br/exportador.aspx?arquivo=PRECO415_BR_ABS.csv',)]
+
+
+for (by, url) in urls:
+    urlretrieve(url, 'data/average_real_estate_prices_by_%s.csv' % by)
```

Comment:
This might crash in non-Unix OS (I think they use `\` instead of `/`). I'd recommend to use `os.path.join(…)` instead:

```python
urlretrieve(url,  os.path.join('data', 'average_real_estate_prices_by_%s.csv' % by))
```